endpoints/openrtb2/auction.go

```diff
@@ -324,6 +324,7 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		PubID:                      labels.PubID,
 		StoredImp:                  adUnitName,
 		AbBucketList:               labels.AbBuckets,
```

Comment:
I think this code is kind of confusing, we have two AbBuckets types in two different package, `labels` and `logMsg`. Can we make the logic more cleaner? 

The raw bucket info is retrieved from AB system, then let's just pass this raw information to down stream logic and down stream logci can do whatever modification they need to do. If we need to get the intersection of buckets defined on AB system and the ones defined locally, then let that part of logic do it.

Reply:
the intersection logic happens before this Auction request, labels is used for grafna metrics at bidder level, logMsg is used for loki and kafka topic